.vib/keycloak/cypress/cypress/integration/keycloak_spec.js

```diff
@@ -0,0 +1,82 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login/logout and user manipulation', () => {
+  cy.login();
+  cy.get('#input-error').should('not.exist');
+  cy.get('.dropdown-toggle').click({ force: true });
+  cy.contains('a', 'Manage account').click({ force: true });
+  cy.contains('p', 'Account console loading').should('not.exist'); //Wait until the loading is finished to continue with actions
+  cy.contains('a', 'Personal').click({ force: true });
+  cy.fixture('user').then((user) => {
+    cy.get('#first-name').clear().type(`${user.newUser.firstName}.${random}`);
+    cy.get('#last-name').clear().type(`${user.newUser.lastName}.${random}`);
+    cy.get('#email-address').clear().type(`${user.newUser.email}`);
+    cy.get('#save-btn').click();
+    cy.contains('.pf-c-alert__title', 'has been updated');
+    cy.get('#signOutButton').click();
+    cy.contains('button#landingSignInButton', 'Sign In');
+  });
+});
+
+it('allows creating a new user', () => {
+  cy.login();
+  cy.get('[data-ng-show="access.queryUsers"]').click();
+  cy.get('#createUser').click();
+  cy.fixture('user').then((user) => {
+    cy.get('#username').type(`${user.newUser.username}.${random}`);
+    cy.get('#email').type(`${user.newUser.email}.${random}`);
+    cy.get('#firstName').type(`${user.newUser.firstName}.${random}`);
+    cy.get('#lastName').type(`${user.newUser.lastName}.${random}`);
+  });
+  cy.contains('button', 'Save').click();
+  cy.contains('.alert', 'Success');
+});
+
+it('allows the upload and delete of a client ', () => {
+  cy.login();
+  cy.contains('Import').click();
+  cy.get('input#import-file').selectFile(
+    'cypress/fixtures/import-client.json',
+    {
+      force: true,
+    }
+  );
+  cy.contains('button', 'Import').click();
+  cy.contains('.alert', 'Success');
```

Comment:
We are actually deleting it, please check the name of the test and the steps that follow.

Reply:
But this could result in failure, though. What would happen if something fails before this specific test ends? The entry would not be removed, and subsequent executions will always fail 🤔

Reply:
I see the problem, although it's very rare this could happen, it is theoretically possible.
A possible way to solve this would be to take the JSON and randomise the data. I forsee this could take me an afternoon to figure out.
Do you think it's worth the effort?

Reply:
I can do an alternative way of uploading a file in this application which lets me randomise the name. 
This still remains a problem in other apps though.
![Screenshot 2022-04-29 at 12 27 06](https://user-images.githubusercontent.com/47592680/165927993-2c6ea533-dab0-485d-b512-f624300f0121.png)

Reply:
I've added the changed test. Please let me know if this is better so I can close the conversation.

Reply:
Indeed, I prefer this alternative. Thanks!